tests/integration/uid_entrypoint.sh

```
@@ -0,0 +1,7 @@
+#!/bin/sh
+if ! whoami &> /dev/null; then
```

Shellcheck (0.11.0)

[warning] In POSIX sh, &> is undefined.

(SC3020)